### ISSUE TITLE
Not working (as of 01/08/2018)

### ISSUE THREAD

**Issue by adam-law** [saghul/jitsi-meet-api#2]
I tried out the example (full), and it doesn't go all the way to create the room. I just get a sea-blue(?) background. I do confirm that it's trying to communicate with JITSI

**Comment by saghul** [comment]
Yeah, we made some API changes so this library will need to be adjusted. I'll do that when I get some time, but PRs are definitely welcome :-)

**Comment by adam-law** [comment]
I wish I can, but I don't know where to start. I've also reviewed the examples on the actual jitsi meet API repository and, it too, isn't working (from what I ran).
 
Edit: I mean if the issue is rooted on the actual API, then I wouldn't know where to start with addressing that.

**Comment by saghul** [comment]
If you have trouble with the official API, please open an issue there, ideally with sample code.

**Comment by adam-law** [comment]
I'm not sure yet, unfortunately. I only ran the example found on the docs portion of their repo. Unlike here, it gave me no indication what was going wrong. At least here, I keep seeing attempts to bind the room via POST, that fails quietly.
